addons/website_project_issue/controllers/main.py

```diff
@@ -58,6 +64,23 @@ def my_issues(self, page=1, date_begin=None, date_end=None, sortby=None, filterb
         if date_begin and date_end:
             domain += [('create_date', '>', date_begin), ('create_date', '<=', date_end)]
 
+        # search
+        if search and search_in:
```

Comment:
If you remove all feature from Odoo, I'm sure that the code will be easier to read !